audits/manifest/icons-192.js

```diff
@@ -32,7 +32,8 @@ class ManifestIcons192 {
 
     if (manifest && manifest.icons && manifest.icons.value) {
       const icons192 = manifest.icons.value.find(icon => {
-        return icon.value.sizes.value.indexOf('192x192') !== -1;
+        const sizesArray = icon.value.sizes.value;
```

Comment:
oh, ha, I thought you were just making fun of the verbosity. If `icon` exists, then `icon.value` and (because of the parsing algorithm) then `icon.value.sizes` exists, but its `value` may be `undefined`[1]. This should probably be tested in the manifest-parser tests...though better would be using a type system for compile-time testing :)

[1] this may be a bug I need to fix to match the spec, and if `icon` exists then when it has no sizes its `sizes.value` property needs to be an empty array